roborazzi/src/main/java/com/github/takahirom/roborazzi/Roborazzi.kt

```diff
@@ -169,6 +173,43 @@ fun captureScreenRoboImage(
   }
 }
 
+@InternalRoborazziApi
+fun captureScreenIfMultipleWindows(
```

Comment:
@sergio-sastre  
Thanks as always. I'm thinking about adding support for the `AlertDialog()` Composable.  
Currently, it throws an `IllegalStateException`, and I would like to fix that behavior.

The basic strategy here is to use `captureScreenRoboImage()` (which merges all windows into one screenshot) if there are multiple windows.
Do you have any thought on this?
<img width="886" alt="image" src="https://github.com/user-attachments/assets/2f08849a-6978-4638-931c-41e7e0a2189b" />

Reply:
@takahirom 
Oh sorry, I‘ve just seen this and also saw that it was merged.
I think it is fine as first solution.

Getting it right with the windows is a tricky one. 

Have you tried by converting the View to bitmap and capturing a bitmap from it? That would likely avoid the IllegalStateException. However, I am not sure whether we could use PixelCopy to draw the bitmap with elevation.

what I mean is to use sth like this `composeView.drawToBitmap.captureRoboImage()`

maybe worth giving it a try?